x-pack/plugins/cloud_security_posture/server/routes/status/status.ts

```diff
@@ -167,12 +167,7 @@ export const getCspStatus = async ({
     checkIndexStatus(esClient, FINDINGS_INDEX_PATTERN, logger, 'kspm'),
     checkIndexStatus(esClient, BENCHMARK_SCORE_INDEX_DEFAULT_NS, logger, 'kspm'),
 
-    checkIndexStatus(
-      esClient,
-      LATEST_VULNERABILITIES_INDEX_DEFAULT_NS,
-      logger,
-      VULN_MGMT_POLICY_TEMPLATE
-    ),
+    checkIndexStatus(esClient, LATEST_VULNERABILITIES_INDEX_DEFAULT_NS, logger, 'all'),
```

Comment:
It would work, but it is unclear why choosing all. I'd suggest fixing `checkIndexStatus` signature and name. Because it starts doing more than what its name suggests.

For example, even by having two different functions: `checkIndexStatus` & `checkIndexStatusByPostureType` makes the code more readable and easy to understand.

Reply:
we can also make the last parameter optional, which don't do any filtering